docs/user-guide/petset/bootstrapping/index.md

```diff
@@ -9,6 +9,7 @@ assignees:
 - smarterclayton
 ---
 
-* Starting in Kubernetes version 1.5, PetSet has been renamed to [StatefulSet](/docs/concepts/abstractions/controllers/statefulsets).
-* This document has been deprecated. For information on working with StatefulSet, see the tutorial on [how to run replicated stateful applications](/docs/tutorials/stateful-application/run-replicated-stateful-application).
+* Starting in Kubernetes version 1.5, PetSet has been renamed to [StatefulSet](/docs/concepts/abstractions/controllers/statefulsets). To use (or continue to use) PetSet in Kubernetes 1.5, you _must_ migrate your exsiting PetSets to StatefulSets. For information on working with StatefulSet, see the tutorial on [how to run replicated stateful applications](/docs/tutorials/stateful-application/run-replicated-stateful-application).
+
+* __This document has been deprecated__, but can still apply if you're using Kubernetes version 1.4 or earlier.
```

Comment:
If we're going to say this document is deprecated but can still apply, then we need to keep the content of the document.